cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -244,4 +253,16 @@ function up() {
     # [1] https://github.com/kubevirt/kubevirtci/issues/906
     # [2] https://github.com/k8snetworkplumbingwg/multus-cni/issues/982
     copy_istio_cni_conf_files
+
+    deploy_kwok
+    until wait_for_kwok_ready; do
+        echo "Waiting for kwok components..."
+        sleep 5
+    done
+}
+
+function wait_for_kwok_ready() {
```

Comment:
nit: Could we move this wait function to be alongside the deploy function?

Reply:
updated, ptal